Framework/API/test/ScopedWorkspaceTest.h

```diff
@@ -22,6 +22,9 @@ class MockWorkspace : public Workspace {
   MockWorkspace *doClone() const override {
```

Comment:
This type should be called `FakeWorkspace` not `MockWorkspace`, but you only extended this type, so I don't suggest fixing that.